connectors/sources/confluence.py

```diff
@@ -304,6 +312,34 @@ def tweak_bulk_options(self, options):
         """
         options["concurrent_downloads"] = self.concurrent_downloads
 
+    async def validate_config(self):
+        """Validates whether user input is empty or not for configuration fields.
+        Also validate, if user configured spaces are available in Confluence.
+
+        Raises:
+            Exception: Configured keys can't be empty
+        """
+        self.configuration.check_valid()
+        await self._remote_validation()
+
+    async def _remote_validation(self):
+        if self.spaces == ["*"]:
+            return
+        space_keys = []
+        try:
+            async for response in self.confluence_client.paginated_api_call(
+                url_name=SPACE, api_query=SPACE_QUERY
+            ):
+                spaces = response.get("results", [])
+                space_keys.extend([space["key"] for space in spaces])
+            if unavailable_spaces := set(self.spaces) - set(space_keys):
+                raise ConfigurableFieldValueError(
+                    f"Spaces '{', '.join(unavailable_spaces)}' are not available. Available spaces are: '{', '.join(space_keys)}'"
+                )
+        except Exception:
```

Comment:
I don't think we need that `except` block here as every exception from `validate_config` are caught on framework level and logged already (see: [Sync Job Runner](https://github.com/elastic/connectors-python/blob/main/connectors/sync_job_runner.py#L161))